lib/tasks/copy_database_to_s3.rake

```diff
@@ -0,0 +1,24 @@
+require 'tty/command'
+require 'down'
+
+namespace :scihist do
+  desc """
+    Copy the latest copy of the database to a location on s3.
+
+    AWS_ACCESS_KEY_ID=joe AWS_SECRET_ACCESS_KEY=schmo bundle exec rake scihist:copy_database_to_s3
+    Note: this is a Postgres binary .dump file, not ASCII SQL. You can convert it to regular sql like this: pg_restore -f mydatabase.sql latest.dump
+  """
+
+  task :copy_database_to_s3 => :environment do
+    cmd = TTY::Command.new(output: Rails.logger)
+    get_database_url_args = ['heroku', 'pg:backups:url']
+    result =  cmd.run!(*get_database_url_args, binmode: true, only_output_on_error: true)
```

Comment:
I don't think there's any benefit to abstract this as `get_database_url_args`, it is easier to read if you just put them right in there:

   cmd.run!("heroku", "pg:backups:url"....

Reply:
I also don't think there's any reason this needs `binmode`?